.drone.yml

```diff
@@ -176,6 +176,14 @@ pipeline:
       branch: [ refs/tags/* ]
       status: success
 
+  notify:
```

Comment:
The push build failed with  `ERROR: Image not found` 

As per the docs http://plugins.drone.io/drone-plugins/drone-downstream/ 
I think you dont need the step `notify:` as per the docs here 

I think you should have it as 

```
pipeline:
... 
...
  trigger-downstream:
    image: plugins/downstream
    server: https://ci.vcna.io
    token: ${DRONE_TOKEN}
    fork: true
    repositories:
       - vmware/vic-product
```